Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -2,6 +2,9 @@ import UIKit
 import Checkout
 
 public typealias Card = Checkout.Card
+public typealias Address = Checkout.Address
+public typealias Phone = Checkout.Phone
+public typealias Country = Checkout.Country
```

Comment:
Can we move these out to a centralised place? There are a good amount of types to re-export and we can also clean up implementation within the Frames module

Reply:
Moved